.github/workflows/CI.yml

```diff
@@ -74,8 +74,8 @@ jobs:
         run: pytest --cov=scripts --cov-branch --cov-report=term-missing
           
       - name: submit coverage report
-        uses: codecov/codecov-action@v3.1.0
+        uses: codecov/codecov-action@v3.1.1
         with:
-          files: ./scripts/tests/coverage
+          files: /home/runner/work/mirflowz/mirflowz/scripts/tests/coverage
```

Comment:
Can you explain why you did this? It's better to have a relative path. With absolute paths, if GitHub Actions changes the default file system organization, our pipeline will break.

Reply:
Oh, okay, I've just read your comment above.